src/resolve/resolved-dns-transaction.h

```diff
@@ -214,11 +214,28 @@ DnsTransactionSource dns_transaction_source_from_string(const char *s) _pure_;
 /* Maximum attempts to send LLMNR requests, see RFC 4795 Section 2.7 */
 #define LLMNR_TRANSACTION_ATTEMPTS_MAX 3
 
-/* Maximum attempts to send MDNS requests, see RFC 6762 Section 8.1 */
-#define MDNS_TRANSACTION_ATTEMPTS_MAX 3
-
-#define TRANSACTION_ATTEMPTS_MAX(p) (((p) == DNS_PROTOCOL_LLMNR) ? \
-                                         LLMNR_TRANSACTION_ATTEMPTS_MAX : \
-                                         (((p) == DNS_PROTOCOL_MDNS) ? \
-                                             MDNS_TRANSACTION_ATTEMPTS_MAX : \
-                                             DNS_TRANSACTION_ATTEMPTS_MAX))
+/* Maximum attempts to send MDNS requests is one except for probe requests, see RFC 6762 Section 8.1 */
+#define MDNS_TRANSACTION_ATTEMPTS_MAX 1
+
+#define MDNS_PROBE_TRANSACTION_ATTEMPTS_MAX 3
```

Comment:
RFC 6762 Section 8.1 says about probe requests. Where can I find that other requests should be tried only once?
Could you update comment more, e.g. quoting a relevant sentence in the RFC?

Reply:
The RFC doesn't explicitly state that a non-probe mDNS query should only be attempted once.
However, since RFC 6762 differentiates between normal (single-shot/continuous) and probe requests, it probably makes sense to fire each normal query only once with no retries.
Otherwise we'd be sending out three attempts for a normal query as well.
The exception of course being probing, which should be attempted a max of 3 times as described in section 8.1.
This also messes with the timing of continuous querying in #22532.
Avahi's implementation is also in line with this.

Reply:
[This blurb](https://www.rfc-editor.org/rfc/rfc6762#page-4:~:text=(Note%20that%20probing%20is%20the%20one%0A%20%20%20exception%20from%20the%20normal%20rule%20that%20there%20should%20be%20at%20least%20one%0A%20%20%20second%20between%20repetitions%20of%20the%20same%20question%2C%20and%20the%20interval%0A%20%20%20between%20subsequent%20repetitions%20should%20at%20least%20double.)) in RFC6762 also points out the difference in timing restrictions for probe and regular queries.

Reply:
I see. That part indeed implies the regular queries should be tried once. Could you update the comment in the code with quoting the above sentences?